src/sentry/static/sentry/app/components/sidebar/help.jsx

```diff
@@ -0,0 +1,110 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import SentryTypes from 'app/sentryTypes';
+import {openDocsSearchModal} from 'app/actionCreators/modal';
+import {t} from 'app/locale';
+import DropdownMenu from 'app/components/dropdownMenu';
+import InlineSvg from 'app/components/inlineSvg';
+import SidebarItem from 'app/components/sidebar/sidebarItem';
+import HookStore from 'app/stores/hookStore';
+
+import SidebarMenuItem from './sidebarMenuItem';
+import SidebarDropdownMenu from './sidebarDropdownMenu.styled';
+
+class SidebarHelp extends React.Component {
+  static propTypes = {
+    orientation: PropTypes.oneOf(['top', 'left']),
+    collapsed: PropTypes.bool,
+    hidePanel: PropTypes.func,
+    organization: SentryTypes.Organization,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      supportMenuItem: null,
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.organization) {
+      HookStore.get('sidebar:contact-support').map(cb =>
```

Comment:
re: calling the hook with SidebarMenuItem, my thinking was only sentry knows what style is needed for this dropdown, so it should be the one telling getsentry that (e.g. we should be able to change SidebarMenuItem to something else in the future without breaking hooks). But seems like this is not the case in organization-dropdown-menu? Nbd, I can change that if you want.

re:handleSupportHookUpdate, this is to avoid having a loading bar as a dropdown item. In a regular setting a loading bar is fine, but it looks weird inside a dropdown. doesn't feel like it's worth having one less param to the hook, but lmk if you think otherwise.

Reply:
Ah I see the Components dict now